api/controller.py

```diff
@@ -289,10 +289,10 @@ def get_domain(url):
             return scheme + "://" + netloc
 
         sitewide_patron_web_client_urls = ConfigurationSetting.sitewide(
-            self._db, Configuration.PATRON_WEB_HOSTNAMES).json_value
+            self._db, Configuration.PATRON_WEB_HOSTNAMES).value
         if sitewide_patron_web_client_urls:
-            for url in sitewide_patron_web_client_urls:
-                patron_web_domains.add(get_domain(url))
+            for url in sitewide_patron_web_client_urls.split(','):
+                patron_web_domains.add(get_domain(url.strip()))
```

Comment:
Two potential issues here:
1. A comma is a valid url value, though an apostrophe/quotation mark is not. It's unlikely they will need to pass in a url with a comma, but I suppose possible. I guess I assumed that processing it into a json list would avoid that by requiring that only quote-surrounded commas be considered delimiters. 
1. If you pass `'"beta.library.org","catalog.library.org"'`, won't we also need to strip the quotation marks from the string before passing it to `get_domain`?
1. Previously I believe you were able to specify `https://catalog.library.org` and thereby disallow `http://catalog.lobrary.org`, though I'm not sure that's the right mechanism for that protection.

Reply:
> Two potential issues here:
> 
>     1. A comma is a valid url value, though an apostrophe/quotation mark is not. It's unlikely they will need to pass in a url with a comma, but I suppose possible. I guess I assumed that processing it into a json list would avoid that by requiring that only quote-surrounded commas be considered delimiters.

Yes, comma is a reserved character in the URI spec, so, while it's unlikely to occur in our URLs, I think we should just avoid it. I would suggest the pipe/vertical bar with any number of spaces around it.

>     2. If you pass `'"beta.library.org","catalog.library.org"'`, won't we also need to strip the quotation marks from the string before passing it to `get_domain`?

My own preference would be URLs without quotes (or other paired delimiters) around each of them.

Reply:
My preferences is also not to quote the string. I thought a comma for a separator was okay because its not allowed in the domain or scheme, but I think it makes sense to modify this to be a pipe character to be cautious. I updated the PR.